File: app/src/main/java/com/hey/mietunoff/mietunofficial/model/news/Enclosure.java
package com.hey.mietunoff.mietunofficial.model.news;

import org.simpleframework.xml.Attribute;

/**
 * Contains a url of news picture
 * Part of {@link Article}
 */

public class Enclosure {

    @Attribute(name = "url")
    private String url;

    @Attribute(name = "length")
    private long length;

    @Attribute(name = "type")
    private String type;

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }

}

File: miet-api/src/main/java/com/alex/miet/miet_api/data/news/Enclosure.kt
package com.alex.miet.miet_api.data.news

import org.simpleframework.xml.Attribute

/**
 * Contains a url of news picture
 * Part of [Article]
 */
class Enclosure {
    @Attribute(name = "url")
    var url: String? = null

    @Attribute(name = "length")
    private val length: Long = 0

    @Attribute(name = "type")
    private val type: String? = null
}